cves/CVE-2016-2327.yml

```diff
@@ -348,7 +367,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This vulnerability is most likely the result of a design oversight, and a lack of testing. The fix for this
+    vulnerability follows the mitigations for this CWE by changing the functionality such that instead of using an
+    unsafe memcpy call, a dedicated function is called to perform the deep copying of a frame, av_frame_copy, that
```

Comment:
I'm wondering if it's about using memcpy, then that's more implementation than design. Or maybe I'm missing something about how the design plays into this. I would say it's about not FOLLOWING the design of using `av_frame_copy`. And lack of testing for sure.

Reply:
I did some more researching, and it seems the av_frame_copy method did in fact exist at the time that the vulnerable code was written. It's hard to say, but I'd guess that the developer should have been using that method in the first place, and forgot or neglected to. I've updated this section to reflect this.